proto/buf.yaml

```diff
@@ -12,9 +12,27 @@ lint:
     # "legacy" lib protos.
     - teleport/lib/multiplexer/test/ping.proto
     - teleport/lib/web/envelope.proto
-    # TODO: Take care of these.
-    - teleport/lib/teleterm
-    - teleport/lib/prehog
+  ignore_only:
+    # RPC_RESPONSE_STANDARD_NAME and RPC_REQUEST_RESPONSE_UNIQUE make it impossible to follow
+    # Google's standard methods. https://cloud.google.com/apis/design/standard_methods
+    RPC_RESPONSE_STANDARD_NAME:
+      - teleport/lib/teleterm
+    RPC_REQUEST_RESPONSE_UNIQUE:
+      - teleport/lib/teleterm
```

Comment:
This ignore used to be in lib/teleterm's lint config, the comment is based on a discussion we had. https://github.com/gravitational/teleport/pull/15877#discussion_r957451049

@codingllama Should I ignore them globally for everything under the `proto` dir?

Reply:
Yep, we can put those in the `except` session, like its api/ counterpart: https://github.com/gravitational/teleport/blob/e1673dec703fe9d1beec67ef39f625c13a9f8663/api/proto/buf.yaml#L15-L19